From f850686d1442e6c4f409f8e24ef461e29710277b Mon Sep 17 00:00:00 2001
From: Zuzana Svetlikova <zsvetlik@redhat.com>
Date: Fri, 17 Apr 2020 12:59:44 +0200
Subject: [PATCH] Disable running gyp on shared deps

Signed-off-by: rpm-build <rpm-build>
---
 Makefile |  2 +-
 node.gyp | 17 -----------------
 2 files changed, 1 insertion(+), 18 deletions(-)

diff --git a/Makefile b/Makefile
index 05fdc1509844ad616e1ae1490bb0f244627fcda9..46816fe1bc7108bd4f0abd6277b2a4f9b4283f95 100644
--- a/Makefile
+++ b/Makefile
@@ -169,7 +169,7 @@ with-code-cache test-code-cache:
 	$(warning '$@' target is a noop)
 
 out/Makefile: config.gypi common.gypi node.gyp \
-	deps/uv/uv.gyp deps/llhttp/llhttp.gyp deps/zlib/zlib.gyp \
+	deps/llhttp/llhttp.gyp \
 	tools/v8_gypfiles/toolchain.gypi tools/v8_gypfiles/features.gypi \
 	tools/v8_gypfiles/inspector.gypi tools/v8_gypfiles/v8.gyp
 	$(PYTHON) tools/gyp_node.py -f make
diff --git a/node.gyp b/node.gyp
index 1885a8d1ecb983fe8fd1ce3cc8cd53ee48e241d2..917eeb73d20791625e71b3e921b2fa9a68fd6133 100644
--- a/node.gyp
+++ b/node.gyp
@@ -428,23 +428,6 @@
               ],
             },
           ],
-         }, {
-           'variables': {
-              'opensslconfig_internal': '<(obj_dir)/deps/openssl/openssl.cnf',
-              'opensslconfig': './deps/openssl/nodejs-openssl.cnf',
-           },
-           'actions': [
-             {
-               'action_name': 'reset_openssl_cnf',
-               'inputs': [ '<(opensslconfig)', ],
-               'outputs': [ '<(opensslconfig_internal)', ],
-               'action': [
-                 '<(python)', 'tools/copyfile.py',
-                 '<(opensslconfig)',
-                 '<(opensslconfig_internal)',
-               ],
-             },
-           ],
          }],
       ],
     }, # node_core_target_name
-- 
2.39.0

