From dc60fa79586e5e8681f019e80080aebc227d0c03 Mon Sep 17 00:00:00 2001
From: Zuzana Svetlikova <zsvetlik@redhat.com>
Date: Thu, 27 Apr 2017 14:25:42 +0200
Subject: [PATCH 1/2] Disable running gyp on shared deps

---
 Makefile | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7f9a57804b77606a3bec03f6be707969d71d3562..dd68e1ba2d3928f46c5d1f8991ded98bc1465323 100644
--- a/Makefile
+++ b/Makefile
@@ -121,14 +121,13 @@ with-code-cache:
 
 .PHONY: test-code-cache
 test-code-cache: with-code-cache
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) code-cache
 
-out/Makefile: common.gypi deps/uv/uv.gyp deps/http_parser/http_parser.gyp \
-              deps/zlib/zlib.gyp deps/v8/gypfiles/toolchain.gypi \
-              deps/v8/gypfiles/features.gypi deps/v8/gypfiles/v8.gyp node.gyp \
-              config.gypi
+out/Makefile: common.gypi deps/http_parser/http_parser.gyp \
+              deps/v8/gypfiles/toolchain.gypi deps/v8/gypfiles/features.gypi \
+			  deps/v8/gypfiles/v8.gyp node.gyp config.gypi
 	$(PYTHON) tools/gyp_node.py -f make
 
 config.gypi: configure
 	@if [ -x config.status ]; then \
 		./config.status; \
-- 
2.19.0.rc1

